BrowserKit/Tests/SiteImageViewTests/SVGImageProcessorTests.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+@testable import SiteImageView
+import SwiftDraw
+import Kingfisher
+
+class SVGImageProcessorTests: XCTestCase {
+    let svgFaviconURL = URL(string: "https://news.ycombinator.com/y18.svg")!
+    let icoFaviconURL = URL(string: "https://www.mozilla.org/favicon.ico")!
+
+    func testDownloadingSVGImage_withKingfisherProcessor() async {
+        let exp = expectation(description: "Image download and parse")
+
+        let siteDownloader = DefaultSiteImageDownloader()
+        siteDownloader.downloadImage(with: svgFaviconURL, options: [.processor(SVGImageProcessor())]) { result in
```

Comment:
Do you really need to test downloading of the image? 🤔 

Since these tests are specific for SVGImageProcessing I wonder if we can get away by simply testing if this successfully supplying a test SVG icon.

Just a thought to avoid downloading as it seems this test is trying to do two things but you only want to do b)
a) Download
b) Process SVG